build.sbt

```diff
@@ -45,7 +45,7 @@ libraryDependencies ++= List(
   "org.scalatest" %% "scalatest" % "3.2.9" % Test
 )
 
-scalaVersion := "2.12.14"
+scalaVersion := "2.12.15"
```

Comment:
except this one which could be merged independently, all other bumps are pending https://github.com/scalacenter/scalafix/pull/1473